File: python/5 kyu/400.py
# https://www.codewars.com/kata/526dbd6c8c0eb53254000110/train/python

# Passed

def alphanumeric(password: str) -> bool:
    result = len(password) > 0 and all(char.isalnum() for char in password)
    return result

output = alphanumeric("PassW0rd")
print(output)

File: nodejs/6 kyu/400.js
// https://www.codewars.com/kata/53046ceefe87e4905e00072a/train/javascript

// Passed

function palindrome(string) {
  let filteredCharacters = Array.from(string)
    .map((char) => char.toLowerCase())
    .filter((char) => {
      let charCode = char.charCodeAt();
      return (
        (charCode >= 97 && charCode <= 122) ||
        (charCode >= 49 && charCode <= 57)
      );
    });

  let result = filteredCharacters.length
    ? [...filteredCharacters].join("") === filteredCharacters.reverse().join("")
    : true;
  return result;
}

const output = palindrome("A man, a plan, a canal - Panama");
console.log(output);